gerd/frontends/generate.py

```diff
@@ -50,6 +50,8 @@ def load_model(model_name: str, origin: str) -> dict[str, Any]:
     model_config.model.name = model_name
     if origin != "None":
         model_config.model.loras.add(lora_dir / origin)
+    else:
```

Comment:
that's not the right change. in line 49 config.model_copy() should be config.model_copy(deep=True)